db/db_impl/db_impl_open.cc

```diff
@@ -1429,6 +1436,22 @@ Status DBImpl::WriteLevel0TableForRecovery(int job_id, ColumnFamilyData* cfd,
   return s;
 }
 
+Status DBImpl::MaybeVerifyFileChecksums() {
+  Status s;
+#if !defined(ROCKSDB_LITE) && defined(OS_LINUX)
```

Comment:
Actually CorruptionTest is failing on macos and probably others. Update the relevant test check to be Linux only (for this patch)?

Reply:
Sure, will enable the failing check only in Linux.

Reply:
MacOS test failure is fixed.